corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -605,6 +608,17 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             };
         },
 
+        onRender() {
+            var submitButton = this.ui.submitButton;
+            if (this.options.sidebarEnabled) {
+                if (JSON.parse(sessionStorage.submitDisabled)) {
```

Comment:
why not `submitButton.prop('disabled', JSON.parse(sessionStorage.submitDisabled));`?